fix: fixed the ArrayIndexOutOfBoundsException

OutOfEastGate opened the pull request:
Fix #311

TraMiu commented:
@OutOfEastGate 
Hi, I have used the MySQL adapter and I generated the policy with null value by INSERT INTO `casbin_rule` directly will null value. 
Later, when I changed the value from null to true, it worked.
I want to help review this PR, but don't know how to add changes to my local Enforcer.java because it is in View Mode. 
Can you show me how can I help you?

OutOfEastGate commented:
Thank you. I have added the code to check for policy, which will no longer throw an array out of bounds, but will give an exception  to prompt the invalid policy. You can check if there are any other questions.

> @OutOfEastGate Hi, I have used the MySQL adapter and I generated the policy with null value by INSERT INTO `casbin_rule` directly will null value. Later, when I changed the value from null to true, it worked. I want to help review this PR, but don't know how to add changes to my local Enforcer.java because it is in View Mode. Can you show me how can I help you?

Thank you. I have added the code to check for policy, which will no longer throw an array out of bounds, but will give an exception  to prompt the invalid policy. You can check if there are any other questions.

OutOfEastGate commented:
> @OutOfEastGate Hi, I have used the MySQL adapter and I generated the policy with null value by INSERT INTO `casbin_rule` directly will null value. Later, when I changed the value from null to true, it worked. I want to help review this PR, but don't know how to add changes to my local Enforcer.java because it is in View Mode. Can you show me how can I help you?

Thank you. I have added the code to check for policy, which will no longer throw an array out of bounds, but will give an exception  to prompt the invalid policy. You can check if there are any other questions.
